nanostack-event-loop/eventOS_event.h

```diff
@@ -61,6 +76,11 @@ typedef struct arm_event_storage {
         ARM_LIB_EVENT_USER,
         ARM_LIB_EVENT_TIMER,
     } allocator;
+    enum {
```

Comment:
Namespacing just occurred to me - these enums end up in global namespace, so should have a prefix like the allocator above.